jme3-core/src/main/java/com/jme3/renderer/Camera.java

```diff
@@ -580,6 +580,57 @@ public void setFrustumTop(float frustumTop) {
         onFrustumChange();
     }
 
+    /**
+     * Obtains field of view when the camera is in perspective mode.
+     *
+     * @return Frame of view angle along the Y in degrees, or 0 if the camera is in orthogonal mode.
+     */
+    public float getFov() {
+        if (!this.parallelProjection)
+        {
```

Comment:
The opening brace (`{`) should go on the same line as the `if`. 3 instances in this file.